controllers/volumereplication_controller.go

```diff
@@ -84,9 +85,16 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		return reconcile.Result{}, err
 	}
 
+	instance.Status.LastStartTime = metav1.Now()
+	if err = r.Client.Update(context.TODO(), instance); err != nil {
+		r.Log.Error(err, "Failed to update status", "Object", instance)
```

Comment:
```suggestion
		r.Log.Error(err, "Failed to update status", "Object", instance.Name)
```

Reply:
I was thinking of dumping the entire object, but we could also go with Name and Namespace.

Reply:
Done